packages/ffe-form-react/src/Tooltip.js

```diff
@@ -53,6 +53,7 @@ class Tooltip extends React.Component {
                     onClick={this.onToggle}
                     type="button"
                     tabIndex={tabIndex}
+                    id={uuid.v4()}
```

Comment:
Maybe we can generate the id in the constructor like we already do with `tooltipId`?

This will generate a new id on every single render  and trigger unnecessary DOM-updates. While running the render method is quite cheap, touching the DOM isn't.

Reply:
That's a good point, I'll create a new fix for that, @ivarni

Reply:
Seems I may have been a bit trigger happy, sorry about that.